cli/lib/kontena/cli/master/config/get_command.rb

```diff
@@ -12,16 +12,14 @@ class GetCommand < Kontena::Command
 
     option ['-p', '--pair'], :flag, "Print key=value instead of only value"
 
-    option '--return', :flag, "Return the value", hidden: true
+    def response
+      client.get("config/#{key}")
+    end
 
     def execute
-      if self.pair?
-        puts client.get("config/#{self.key}").inspect
```

Comment:
it used to output a hash inspect instead of what the help text implies. `{ "key" => "value" }`